### PR TITLE
Move IstioConfigDetails emptyState to PF4

### PR DESCRIPTION
**Pull request by aljesusg**
Screenshot from 2019-11-18 09-57-05

This blocks migration and https://github.com/kiali/kiali/issues/1857

### CODE REVIEW

File: src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx [hunk 1]
```diff
@@ -442,13 +438,15 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
       // In theory it shouldn't enter here, but we show a nice error page anyway.
 
       return (
-        <EmptyState className={emptyStateStyle}>
-          <EmptyStateIcon name="error-circle-o" />
-          <EmptyStateTitle>Error loading object `{this.props.match.params.object}`</EmptyStateTitle>
-          <EmptyStateInfo>
+        <EmptyState variant={EmptyStateVariant.full}>
+          <EmptyStateIcon color={PFAlertColor.Danger} icon={KialiIcon.Error} />
+          <Title headingLevel="h5" size="lg">
```
Comment (xeviknal) [inline src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx]: nitpick: should level and size would be a constants instead? TitleLevel.h5 and TitleSize.lg?

**Comment by lucasponce** [comment]
Is red -> black font color consistent with other pages ?

File: src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx (hunk 1 above)
Comment (aljesusg) [reply src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx]: We should move this to a general file

**Comment by lucasponce** [comment]
ok

**Pull request merged**